### ISSUE TITLE
[Accessibility] Allow users to change the default location of "GitHub Copilot: Explain This" 

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#204143]
Type: <b>Bug</b>

"GitHub Copilot: Explain This" command always sends the question to the sidebar chat view, which does not have any default keybinding. From a screen reader user's perspective, however, sending this to quick chat is preferred, which has Ctrl+Shift+I default keybinding.

`{"workbench.commandPalette.experimental.askChatLocation": "quickChat"}` setting may cover this adjustment as well. Currently, that setting only covers the questions coming from the command palette input questions.

VS Code version: Code - Insiders 1.87.0-insider (2af613979f646fc4dcebfeaedc7d14f138c7b072, 2024-02-01T23:51:02.394Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-1145G7 @ 2.60GHz (8 x 2611)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.71GB (5.27GB free)|
|Process Argv|--crash-reporter-id b05b88e5-8894-4031-ae34-fa034ebddea9|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (119)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-openapi|42C|4.21.1
android-dev-ext|ade|1.4.0
aiprm-lang|AIP|0.0.2
Bookmarks|ale|13.4.2
openscad|Ant|1.2.2
spellright|ban|3.0.122
zoterolatex|bna|0.4.1
mermaid-markdown-syntax-highlighting|bpr|1.6.0
doxdocgen|csc|1.4.0
vscode-markdownlint|Dav|0.53.0
vscode-eslint|dba|2.4.4
vscode-quick-select|dba|0.2.9
vscode-deno|den|3.33.3
gitlens|eam|14.6.1
EditorConfig|Edi|0.16.4
prettier-vscode|esb|10.1.0
vscode-firefox-debug|fir|2.9.10
shell-format|fox|7.2.5
vscode-google-translate|fun|1.4.13
codespaces|Git|1.16.9
copilot|Git|1.156.686
copilot-chat|Git|0.13.2024020201
remotehub|Git|0.62.0
vscode-github-actions|git|0.26.2
vscode-pull-request-github|Git|0.81.2024020111
cloudcode|goo|2.3.0
overleaf-workshop|iam|0.9.0
cslpreview|igo|0.2.2
easy-snippet|inu|0.7.5
path-autocomplete|ion|1.25.0
latex-workshop|Jam|9.18.0
lilypond-syntax|jea|0.1.1
scheme|jea|0.2.0
better-cpp-syntax|jef|1.17.2
commitlint|jos|2.4.7
language-julia|jul|1.66.2
google-search|kam|0.0.1
vscode-lua-format|Koi|1.3.8
lilypond-formatter|lhl|0.2.3
lilypond-pdf-preview|lhl|0.2.8
lilypond-snippets|lhl|0.1.1
vslilypond|lhl|1.7.3
zotero|mbl|0.1.11
git-graph|mhu|1.30.0
azure-dev|ms-|0.8.0
vscode-azureappservice|ms-|0.25.1
vscode-azurecontainerapps|ms-|0.6.1
vscode-azurefunctions|ms-|1.13.2
vscode-azureresourcegroups|ms-|0.8.3
vscode-azurestaticwebapps|ms-|0.12.2
vscode-azurestorage|ms-|0.15.3
vscode-azurevirtualmachines|ms-|0.6.5
vscode-cosmosdb|ms-|0.20.1
vscode-docker|ms-|1.28.0
vscode-edge-devtools|ms-|2.1.4
black-formatter|ms-|2024.1.10331006
debugpy|ms-|2024.0.0
flake8|ms-|2023.10.0
isort|ms-|2023.13.10231012
python|ms-|2023.25.10221012
vscode-pylance|ms-|2023.12.103
jupyter|ms-|2024.1.2024011201
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.17
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.339.0
remote-ssh|ms-|0.108.2024012417
remote-ssh-edit|ms-|0.86.0
remote-wsl|ms-|0.81.8
vscode-remote-extensionpack|ms-|0.25.0
azure-account|ms-|0.11.6
azure-repos|ms-|0.38.0
cmake-tools|ms-|1.16.32
cpptools|ms-|1.19.2
cpptools-extension-pack|ms-|1.3.0
js-debug-nightly|ms-|2024.1.2217
powershell|ms-|2024.3.0
remote-explorer|ms-|0.5.2024011009
remote-repositories|ms-|0.40.0
remote-server|ms-|1.6.2024011109
vscode-github-issue-notebooks|ms-|0.0.130
vscode-node-azure-pack|ms-|1.2.0
vscode-selfhost-test-provider|ms-|0.3.25
vscode-serial-monitor|ms-|0.11.0
vscode-speech|ms-|0.5.2024020116
vsliveshare|ms-|1.0.5905
windows-ai-studio|ms-|0.2.3
autodocstring|njp|0.6.1
pandocciter|not|0.10.4
shiny-python|Pos|0.1.5
shinyuieditor|pos|0.5.0
quarto|qua|1.110.1
r-debugger|RDe|0.5.4
java|red|1.27.0
vscode-xml|red|0.26.1
vscode-yaml|red|1.14.0
r|REd|2.8.2
multi-command|ryu|1.6.0
AudioQ|Seh|0.0.2
vscode-deepl|soe|1.0.10
abc-music|sof|0.4.0
lua|sum|3.7.4
latex-utilities|tec|0.4.10
cmake|twx|0.0.17
vscode-terminal-here|Tyr|0.2.4
windows-terminal|Tyr|0.7.0
errorlens|use|3.16.0
intellicode-api-usage-examples|Vis|0.2.8
vscodeintellicode|Vis|1.2.30
vscode-conventional-commits|viv|1.25.0
vscode-arduino|vsc|0.6.230727001
vscode-java-debug|vsc|0.56.0
vscode-java-dependency|vsc|0.23.4
vscode-java-pack|vsc|0.25.15
vscode-java-test|vsc|0.40.1
vscode-maven|vsc|0.44.0
markdown-all-in-one|yzh|3.6.2
grammarly|znc|0.25.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vscod805cf:30301675
vsaa593:30376534
py29gd2263:30784851
c4g48928:30535728
a9j8j154:30646983
962ge761:30841072
pythongtdpath:30726887
welcomedialog:30812478
pythonidxpt:30768918
pythonnoceb:30776497
asynctok:30898717
dsvsc013:30777762
dsvsc014:30777825
dsvsc015:30821418
pythontestfixt:30866404
pythonregdiag2:30926734
pyreplss1:30879911
pythonmypyd1:30859725
pythoncet0:30859736
pythontbext0:30879054
accentitlementst:30870582
dsvsc016:30879898
dsvsc017:30880771
dsvsc018:30880772
8082a590:30953407
edj9j583:30943796
fegfb526:30952798
7j2b6412:30951517
e3gdj431:30946825
ccp2r3:30950906

```

</details>

**Comment by meganrogge** [comment]
@jooyoungseo, when I use that command, the chat input gets focused, so a user doesn't have to navigate there. 

However, I agree with you that putting a user's focus there, with no way to get back via a default keybinding, is not very useful. 

Even if we were to make it such that the quick chat opens instead, the question would remain: how do we communicate the way for a user to get back there?
